### ISSUE TITLE
Few improvments

### ISSUE THREAD

**Issue by PxlSyl** [timlrx/tailwind-nextjs-starter-blog#777]
Hi! 

While working on the i18n version, I noticed a few things. These are really minor improvments that being said!

- Draft feature is no more working

-In tags page, the first letter should be capitalized (I think, but perhaps it's a choice)

For the draft feature, it's just a few line of codes to add -with js filtering for the posts- in home page and in blog main page, something like this :   const filteredPosts = posts.filter((post) => post.draft === false) and then render filteredPosts 

For capitalized tags, in the [tag] page, you can do something like this : 

 function capitalizeFirstLetter(str: string): string {
    return str.charAt(0).toUpperCase() + str.slice(1)
  }
  const capitalizedTag = capitalizeFirstLetter(tag)
  return genPageMetadata({
    title: capitalizedTag,
...

**Comment by PrinOrange** [comment]
In fact, for capitalization, yo can do it better with CSS `text-transform` property, [here is the description](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform).
```css
.capitalize{
        text-transform: capitalize;
}
```
It can rectify to capitalize the first letter in word automatically, and the browser would recognize the words to determine what word should be capitalized and what is not necessary.

**Comment by PxlSyl** [comment]
> In fact, for capitalization, yo can do it better with CSS `text-transform` property, [here is the description](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform).
> 
> ```css
> .capitalize{
>         text-transform: capitalize;
> }
> ```
> 
> It can rectify to capitalize the first letter in word automatically, and the browser would recognize the words to determine what word should be capitalized and what is not necessary.

Yeah but thing is, it's for metadata generation and it goes directly to the tab

**Comment by PxlSyl** [comment]
Hi! I also added a share component, and the support for post series, if anyone is interested!
